libs/Common/Octree.inl

```diff
@@ -25,16 +25,23 @@ inline TOctree<ITEMARR_TYPE,TYPE,DIMS,DATA_TYPE>::CELL_TYPE::CELL_TYPE()
 template <typename ITEMARR_TYPE, typename TYPE, int DIMS, typename DATA_TYPE>
 inline TOctree<ITEMARR_TYPE,TYPE,DIMS,DATA_TYPE>::CELL_TYPE::~CELL_TYPE()
 {
-	delete[] m_child;
+    if (m_child != NULL)
```

Comment:
if not needed, and setting it to null neither, as the object is destructed anyway, revert

Reply:
Do different compilers behave the same way?

Reply:
yes